src/css/layout/fixedDataTableCellGroupLayout.css

```diff
@@ -15,7 +15,7 @@
   overflow: hidden;
   position: absolute;
   top: 0;
-  white-space: nowrap;
+  white-space: normal;
 }
 
```

Comment:
Change this style so that the content should not overflow from a particular cell .The text wraps to the next line when needed.